v2/concordium/service.proto

```diff
@@ -89,4 +89,23 @@ service Queries {
   // Run the smart contract entrypoint in a given context and in the state at
   // the end of the given block.
   rpc InvokeInstance (InvokeInstanceRequest) returns (InvokeInstanceResponse);
+
+  // Get the pool delegators of a given pool at the end of a given block.
```

Comment:
It would be good to add a bit of clarification as to the difference between the two different types, the "" and "RewardPeriod" queries.

Maybe the simplest way to express it would be to explain that the "RewardPeriod" ones are the ones whose stake contributes to the baking in the reward period the block is in.